boot/zephyr/main.c

```diff
@@ -138,7 +136,7 @@ void main(void)
 
     BOOT_LOG_INF("Bootloader chainload address offset: 0x%x",
                  rsp.br_image_off);
-    zephyr_flash_area_warn_on_open();
```

Comment:
This was a useful feature that let us notice whenever open/close calls were mismatched; why was it removed?

I know close is currently a no-op but with names like open and close we really should not assume it will remain that way, IMO.

Reply:
As you noticed the close is no-op. And therefore Zephyr doesn't implement longing feature for non-existing operation. Think we can re-implement similar feature when there will be a need for that.

In current code it is still possible to add code for counting open/close operation together - it will
require implementation of wrappers for flash_area_open/close into another mcuboot shim which I don't like.

Reply:
I suggest opening an issue to re-implement when `close` is not a "no-op" anymore and moving.

Reply:
The issue created here: https://github.com/runtimeco/mcuboot/issues/280

Reply:
@mbolivar can you approve?